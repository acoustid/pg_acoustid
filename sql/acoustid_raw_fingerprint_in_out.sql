CREATE EXTENSION IF NOT EXISTS acoustid;

SELECT '{994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}'::acoustid_raw_fingerprint;

SELECT CAST('{994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}'::acoustid_raw_fingerprint AS int4[]);
SELECT CAST('{994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}'::acoustid_raw_fingerprint AS text);

SELECT extract_terms('{994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}'::acoustid_raw_fingerprint);
