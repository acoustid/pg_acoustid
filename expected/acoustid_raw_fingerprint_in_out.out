CREATE EXTENSION IF NOT EXISTS acoustid;
NOTICE:  extension "acoustid" already exists, skipping
SELECT '{994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}'::acoustid_raw_fingerprint;
                                   acoustid_raw_fingerprint                                   
----------------------------------------------------------------------------------------------
 {994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}
(1 row)

SELECT CAST('{994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}'::acoustid_raw_fingerprint AS int4[]);
                                             int4                                             
----------------------------------------------------------------------------------------------
 {994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}
(1 row)

SELECT CAST('{994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}'::acoustid_raw_fingerprint AS text);
                                             text                                             
----------------------------------------------------------------------------------------------
 {994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}
(1 row)

SELECT extract_terms('{994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}'::acoustid_raw_fingerprint);
                                        extract_terms                                         
----------------------------------------------------------------------------------------------
 {994277718,440498550,440498550,180443510,180476246,180485494,193052022,193052006,1233231206}
(1 row)

